### PR TITLE
Change the well being icon

### PR DESCRIPTION
**Pull request by Diama1**
#665

### CODE REVIEW

**Comment by Diama1** [comment]
Screenshot from 2023-03-08 13-40-19

File: client/src/elm/Pages/MessagingCenter/View.elm [hunk 1]
```diff
@@ -938,7 +938,7 @@ messageOptionsDialog language currentTime currentDate nurseId tab state =
                         [ class "ui fluid button cyan"
                         , onClick <| ToggleMessageRead nurseId messageId message isRead
                         ]
-                        [ img [ src "assets/images/envelope.svg" ] []
+                        [ img [ src "assets/images/wellbeing.svg" ] []
```
Comment (anvmn) [inline client/src/elm/Pages/MessagingCenter/View.elm]: Why?

**Review by anvmn — CHANGES_REQUESTED** [review]

File: client/src/assets/images/envelope.svg
Comment (anvmn) [inline client/src/assets/images/envelope.svg:1]: @Diama1 Can you revert this please. There should be no changes here.

Comment (Diama1) [reply client/src/assets/images/envelope.svg:1]: @anvmn  I deleted it in the previous commit then when you gave a comment, I realized that it was being used some where else then I put it back

**Review by anvmn — APPROVED** [review]

**Pull request merged**